samples/bluetooth/peripheral_dis/src/main.c

```diff
@@ -44,6 +45,69 @@ static struct bt_conn_cb conn_callbacks = {
 	.disconnected = disconnected,
 };
 
+
+static int zephyr_settings_fw_load(struct settings_store *cs);
+
+static const struct settings_store_itf zephyr_settings_fw_itf = {
+	.csi_load = zephyr_settings_fw_load,
+};
+
+static struct settings_store zephyr_settings_fw_store = {
+	.cs_itf = &zephyr_settings_fw_itf
+};
+
+static int zephyr_settings_fw_load(struct settings_store *cs)
+{
+	bt_addr_le_t addr;
+
+	addr.type = BT_ADDR_LE_PUBLIC;
+
+	static u8_t addr_val[6] = { 1, 2, 3, 4, 5, 6 };
+	static u8_t irk[16] = { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9,
+				0xa, 0xb, 0xc, 0xd, 0xe, 0xf };
+
+	memcpy(addr.a.val, addr_val, sizeof(addr_val));
+	settings_runtime_set("bt/id", &addr, sizeof(bt_addr_le_t));
```

Comment:
Any good reason for doing this? This should be removed in my opinion.

Reply:
As settings are enabled now this will provide the a backend for reading.

Reply:
But why do you need to store an IRK and Identity Address in settings? That part I don't understand.

Reply:
This is also unclear to me. Furthermore, we don't support storing/restoring of a public address - only static random address, so this would fail later either way.

Reply:
Agreed. I will remove the non DIS parts.